app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,24 @@
+!!!
+%html
```

Comment:
Lol. I was like "why did you delete the application layout?" only to see that you switched to HAML.